deep-learning/_posts/2018-11-09-Incremental-Learning.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: review
+title:  "Incremental Learning for Semantic Segmentation of Large-Scale Remote Sensing Data"
+tags:   Incremental learning, catastrophic forgetting, deep-learning, segmentation, remote-sensing,
+author: Charles Authier
+pdf:  https://arxiv.org/abs/1810.12448
+cite:
+  authors: "Onur Tasar, Yuliya Tarabalka, Pierre Alliez"
+  title:   "Incremental Learning for Semantic Segmentation of Large-Scale Remote Sensing Data"
+  venue:   "Arxiv, 29 oct 2018"
+---
+
+Code: https://project.inria.fr/epitome/inc_learn/
+
+## Method
+One big probleme, catastrophic forgetting: a significant performance drop for the already learned classes when new classes are added on the data, having no annotations for the old classes.
+They propose an incremental learning methodology, enabling to learn segmenting new classes without hindering dense labeling abilities for the previous classes, although the entire previous data are not accessible.
+
+The key points of the proposed approach are adapting the network to learn new as well as old classes on the new training data, and allowing it to remember the previously learned information for the old classes.
```

Comment:
The proposed approach adapts the network to learn....
and allows it to remember